src/Api/Resource/TpayChannel.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommerceWeavers\SyliusTpayPlugin\Api\Resource;
+
+class TpayChannel
+{
+    public function __construct(
+        private ?string $id,
+        private ?string $name,
+        private ?string $fullName,
+        private ?array $image,
+        private ?bool $available,
+        private ?bool $onlinePayment,
+        private ?bool $instantRedirection,
+        private ?array $groups,
+        private ?array $constraints,
```

Comment:
Why not `public readonly`? We can get rid of getters/setters, as they don't seem to be needed in this case, at least for me.